tools/depends/configure.ac

```diff
@@ -281,6 +292,11 @@ case $host in
     if test "x$use_cpu" = "xauto"; then
       use_cpu=$host_cpu
     fi
+    if [[ $host =~ "arm64-" ]] || [[ $host =~ "aarch64-" ]]; then
+      use_cmake_cpu=aarch64
+    else
+      use_cmake_cpu=arm
```

Comment:
For linux I set it to `arm` alone, not sure how there could be easiest (without big checks) the right defined.

Reply:
There is a variable which already contains exactly one of the two values, depending on the host used to configure. So why not use that one?

Reply:
You mean the `arm*` from `arm*-*-linux-gnu*`, can look to extract them from `host`, does it then match the mostly on cmake used variables?

Reply:
The another thing it can also `aarch32` which then not correct to cmake.

Reply:
I add this:
```bash
    if [[ $host =~ "arm64-" ]] || [[ $host =~ "aarch64-" ]]; then
      use_cmake_cpu=aarch64
    elif [[ $host =~ "arm32-" ]] || [[ $host =~ "aarch32-" ]]; then
      # aarch32 related to https://en.wikipedia.org/wiki/ARM_architecture#AArch32 is v7-a
      # arm32 should never be there, only in the check to avoid possible wrong cmake CPU
      use_cmake_cpu=armv7-a 
    else
      use_cmake_cpu=$(echo $host | sed -e 's/\-.*$//')
    fi
```

Then the cpu becomes taken from host, if not `arch*` or `32|64`